lib/apiservers/engine/proxy/container_proxy.go

```diff
@@ -133,11 +133,15 @@ func NewContainerProxy(plClient *client.PortLayer, portlayerAddr string, portlay
 // returns:
 //	(handle string, error)
 func (c *ContainerProxy) Handle(ctx context.Context, id, name string) (string, error) {
+	op := trace.FromContext(ctx, "Handle: %s", name)
```

Comment:
Don't know how broadly used `FromContext` is but maybe we should have this return the ID so we can say:
`op, opID := trace.FromContext(...)`